setup/web_remember_tree_column_width/setup.py

```diff
@@ -0,0 +1,6 @@
+import setuptools
+
+setuptools.setup(
```

Comment:
why this files are need?

Reply:
@hugho-ad this is autogenerated by the bot